doc/source/whatsnew/v0.24.0.txt

```diff
@@ -10,6 +10,27 @@ New features
 
 .. _whatsnew_0240.enhancements.other:
 
+``ExtensionArray`` operator support
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+An ``ExtensionArray`` subclass consisting of objects that have arithmetic and
```

Comment:
Can you include most of this in `extending.rst` as well?

It'd be good to mention the performance implications of using these mixins.